pkg/cri/server/container_create_linux_test.go

```diff
@@ -2259,3 +2311,113 @@ containerEdits:
 		})
 	}
 }
+
+// TestLinuxVolumeMounts tests the linux-specific parts of VolumeMounts.
+func TestLinuxVolumeMounts(t *testing.T) {
+	testContainerRootDir := "test-container-root"
+	idmap := []*runtime.IDMapping{
+		{
+			ContainerId: 0,
+			HostId:      100,
+			Length:      1,
+		},
+	}
+
+	for _, test := range []struct {
+		desc              string
+		criMounts         []*runtime.Mount
+		imageVolumes      map[string]struct{}
+		usernsEnabled     bool
+		expectedMountDest []string
+		expectedMappings  []*runtime.IDMapping
```

Comment:
Would you mind to add test for relative path?

Reply:
Oh, yes, great point. Added now!